packages/jest-validate/README.md

```diff
@@ -0,0 +1,92 @@
+# jest-validate
+
+Generic configuration validation tool that helps you with warnings, errors and deprecation messages as well as showing users examples of correct configuration.
+
+```
+npm install --save-dev jest-validate
+```
+
+## Usage
+
+```js
+import {validate} from 'jest-validate';
+
+validate(
+  config: Object,
+  validConfigSample: Object,
+  deprecatedConfig: ?Object,
+  options: ?Object
+);
```

Comment:
Actually, thinking more about this, how about:

```
type Title = {
  deprecation: string,
  error: string,
  warning: string,
}
```

it seems like some of these should be optional, no? Basically I'd encourage you to think about what is the simplest way this API can be useful to people but still give people the option to express certain things in a powerful way. The motto is "Make easy things easy, make hard things possible".